ratpack/src/main/java/com/baeldung/model/Movie.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.model;
```

Comment:
Just thinking ahead a bit, would you please put these in something like `com.baeldung.ratpack.intro.model` just so that if we do other articles on ratpack, then other authors can get the namespace isolation they need?